src/main/java/com/jadventure/game/monsters/Bugbear.java

```diff
@@ -7,17 +7,22 @@
  */
 public class Bugbear extends Monster {
 	public Bugbear(int playerLevel){
-		this.monsterType = "Bugbear";
+		monsterType("Bugbear");
```

Comment:
I don't quite get the purpose of removing this single line of code into a method.